data/json/items/comestibles/veggy_dishes.json

```diff
@@ -49,6 +49,91 @@
     "vitamins": [  ],
     "flags": [ "EDIBLE_FROZEN" ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "garlic_roasted",
+    "looks_like": "garlic_clove",
+    "name": { "str_sp": "roasted garlic" },
+    "weight": "18 g",
+    "color": "white",
+    "spoils_in": "180 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -1,
+    "healthy": 1,
+    "calories": 36,
```

Comment:
You'll either need to set this to 151 calories or use a flag to ignore ingredients when calculating calories for this item.  It's generally preferred to do the former than the flag.

Reply:
Most of the oil would indeed be absorbed, but for the recipe you'd need to use about half as much volume as currently used, but it can't be set to use less as it's already using just one unit.
I decided to just set the item to have 151 calories.